source/common/config/address_json.cc

```diff
@@ -16,6 +16,13 @@ void AddressJson::translateAddress(const std::string& json_address, bool url, bo
     if (instance->type() == Network::Address::Type::Ip) {
       address.mutable_socket_address()->set_address(instance->ip()->addressAsString());
       address.mutable_socket_address()->set_port_value(instance->ip()->port());
+      if (url) {
```

Comment:
This file is only used IIRC in v1 -> v2 conversion. I think we can ignore UDP for v1 configs since all of this will go away soon?

Reply:
I think I needed it for the test case added to ListenerManagerImplWithRealFiltersTest, since the test cases there use json to specify their listeners.  Would it be better to remove the translation code here and modify the test case in listener_manager_impl_test.cc to not use parseListenerFromJson()?

Reply:
Yeah if you could make that test use v2 config and remove this code ,that would be A++.

Reply:
All set.